MobileFilter/sections/specific_app.txt

```diff
@@ -291,6 +291,8 @@
 ||graph.facebook.com^$app=com.teeonsoft.ztorrent
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74852
+||firebaseremoteconfig.googleapis.com/v1/projects/*/namespaces/firebase:fetch?key=*$app=video.player.videoplayer
```

Comment:
What the purpose of this rule?

Reply:
Ad banner wont load with this URL blocked and HTTPS filtering enabled on my device, but it seems that it doesn't work on user side.